### DISCUSSION TITLE
2023-11-10 - Presentations - Text on slides - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#7211] [Chewing the Fat]
Hey presenter 👋

Leave your thoughts on
https://www.ssw.com.au/rules/no-text-slides/

**Comment by drwharris** [comment]
One of the biggest impediments to good presentations I feel is access to a good library of artwork.

**Reply by brydeno** [reply]
I agree completely.

**Reply by sethdaily** [reply]
Have you used DALLE-3 yet? It can be an easy way to get good artwork without having to search, and it's quick

**Comment by 2fernandez** [comment]
"Less is more" is always a great principle to follow in a presentation.

**Comment by matt-goldman** [comment]
People fall into the trap of thinking their PowerPoint slideshow _is_ their presentation. It's not. PowerPoint is a tool - and it can be an awesome one if used properly. Use it to support your presentation. Your slides should contain visual aids that help to underscore or reinforce the information _you_ are presenting. They should not _be_ the information.

**Comment by bradystroud** [comment]
They should teach this at school

**Comment by leehawkinsssw** [comment]
The rule is a  good reminder to reduce/remove text on slides. It's been a challenge for me to strip down my slides from being very text heavy but I've become much better over time. 

Hat tip to @apforsyth for his idea of pasting all that bullet point content into the speaker notes - then you've still got all that detail to refer to (and to remind you what to cover) but you're not showing it to distract your audience.

**Comment by ozdemir-mehmet** [comment]
Good tips to follow in the rule

**Comment by SamBWagner** [comment]
I think the main point of the talk was 'creating memory tags' and it's only mentioned for a few seconds. Augmentative media should be there to help the audience keep up and learn so doing this is paramount. If you've got information on the slide which is just a regurgitation of what's being said, or worse, just there to help you remember your talk. You'd be better off deleting it.

Moreover, the Rule then proceeds to primarily just give the prospective presenter options for hiding what is still just a bullet point slide. I think more fundamental tips like:

- Build your talk without a slide-deck and then add slides where your explanation is insufficient or visual medium would give the listener more opportunities to understand. This feels like a cart before the horse type of issue.

**Reply by leehawkinsssw** [reply]
> Build your talk without a slide-deck

This is sage advice. Focus on the story you're trying to tell and its key messages first. I generally use a mindmap to help me build out what I want to cover and then create slides later. There's a danger of getting sucked into finessing your slides rather than focusing on the content if you choose to build within slides from the get go.

**Comment by leahy268** [comment]
I have watched a lot of presentations and certainly know the difference between a good one and a bad one.  The same thing can be said with speakers TBH.  Some are awesome, some are not.  
I haven't used PowerPoint very often myself but will remember to use these tips when I do.

**Comment by YalalCakirSSW** [comment]
The pain lies in recognizing that people can't effectively read and listen at the same time, and it's crucial to strike a balance between visual aids and spoken content.  With the help of the PowerPoint's Design Ideas and SmartArt we can create visually appealing slides.

**Comment by jakebayliss** [comment]
Will try to use these next time I make a PowerPoint (which is rarely)

**Comment by joshbermanssw** [comment]
Being able to slowly present information on your slides via animations was crucial to the success of many presentations i saw during the brain storming day. Great rule too!

**Comment by cvionnet** [comment]
Love the idea of keeping it simple.
Some Powerpoint presentations are amazing, however in most of the case it turns into a showcase of animations and designs.
"Look, I've clicked on all the buttons!"

**Comment by bahjatalaadelssw** [comment]
Great idea, and we can add a "no go" zone for the PIP so we don't lose any info.

**Comment by piers-sinclair** [comment]
Writing fewer words is an art form, and often much harder than writing pages of content....and is also a lot more valuable.

**Comment by AlexRothwell** [comment]
I'm glad to see new tools being released to make improving styling and simplicity of slideshows. I look forward to using them next time I'm putting together a presentation!

**Comment by Ben0189** [comment]
When making slides, I always avoid long sentences because people usually won't read them. Instead, I prefer using a few words or pictures so that people can stay engaged even if they lose focus.

**Comment by jeoffreyfischer** [comment]
Totally agree with the rule! And I never heard about SmartArt

**Comment by jimmy1992abc** [comment]
It is amazing to see how the tool works to quickly visualise these bullet points in a presentation. 
The PowerPoint should be a supplement, giving the audience a sense of emotion, and should be avoided to draw the audience's attention from the speaker.

**Comment by zacharykeeping** [comment]
Some good tips here. I don't use PowerPoint often so I hadn't known about some of these features, like SmartArt.

**Comment by chrisschultzssw** [comment]
I don't make a lot of PowerPoints - I hadn't seen the SmartArt feature, I'll definitely give it a go next time.

**Comment by ricksu978** [comment]
Doing animation is straight forward.

I have not used Smart Art before, but I would wonder how to utilize Smart Art while maintaining the style consistency.

**Comment by Dhruv-0987** [comment]
my goal is always to talk fluently (write stuff in the notes section) and have an image or animation for every broader talking I am going to speak on so very less words.

**Comment by tomek-i** [comment]
I think it depends what we do with the slides afterwards.

Many speakers share their slides with the audience afterwards and if the slides are just visuals then there is no context to them anymore because we delivered it verbally during the presentation.

**Comment by Harry-Ross** [comment]
Really important topic - important to be mindful when creating presentations to not have massive walls of text.

**Comment by vladislav-kir** [comment]
"Less is More" is a great rule to follow. 

I have not used SmartArt before, so I am looking forward to trying it, next time I need to make a presentation.

**Comment by tombui99** [comment]
If only these cool new tools such as SmartArt had been around earlier it would have changed 90% of how my previous Powerpoint presentation looked!

**Comment by brydeno** [comment]
Keep your slides super simple. The key is understanding your content well. This is of course often harder than it sounds :)

**Comment by amankumarrr** [comment]
I was unaware of the Smart Art feature until now, and I thoroughly enjoyed the video. The presenter did an excellent job delivering the message with humor.
Using fewer words accompanied by visuals tends to make a more significant impact on the audience.

**Comment by bettybondoc** [comment]
Agree with this rule 100%! When watching a presentation, I always lose interest the moment I see a wall of text. On the extreme end though, I've seen presentations with pictures and graphics that have no relation whatsoever to their topics/slides. Mad respect to great presenters! (Good note on having a separate document for sending afterwards as I reckon that's the main reason why A LOT of people use walls of texts and bullet points)

The SmartArt feature is a welcome surprise for me — feeling a bit silly for all the times I had to make the graphics myself when PowerPoint can churn them out in seconds.

**Comment by divgul** [comment]
Good rule @bradystroud and @GordonBeeming. Loved the video. Thanks for sharing about the Designer and SmartArt. So many PowerPoint presentations are textual and boring. Making them visual makes a lot of sense. It's good to know about the tools available to help with this, but it still won't teach us *how* to make a visual presentation. It's quite an art. But this rule and this CTF is a great start. Glad to see that we as a company are emphasizing this.

**Comment by UlyssesMaclaren** [comment]
Smart art is my key takeaway from this CTF. I'll use it next time

**Comment by jackreimers** [comment]
The rule and video has some great points that I hadn't really considered

**Comment by apforsyth** [comment]
Great video, cleverly showing how images can illustrate your points.  Some good info here for people who want to know more:
https://waterbearlearning.com/mayers-principles-multimedia-learning/

**Comment by tkapa** [comment]
Less words is always nice, but access to whatever replaces them is difficult to get. The slide designer seems useful, but isn't applicable to all scenarios.

**Comment by synnoveisabel** [comment]
Great tips! SmartArt really makes a presentation more entertaining.

**Comment by sethdaily** [comment]
Let the talking do the talking

**Comment by steven0x51** [comment]
SmartArt is amazing, I'll use it more.

**Comment by MattParkerDev** [comment]
Great discussion on presentation efficiency. Emphasizing visuals over text can really boost audience engagement. Excited to apply these tips!

**Comment by stephanfakossw** [comment]
Interesting topic... Agree that good visuals to support the spoken presentation is much better then bullet point. However, finding good visuals might be a very time consuming task so there has to be balance.

**Comment by KostaMadorsky** [comment]
I will definitely give the SmartArt a go.

**Comment by ae-cher-ssw** [comment]
I don't think reducing text on slides makes a presentation great. It's the content. The idea. 

In other words, focus on re-thinking different ways to communicate, as opposed to reducing text or design.

When she was young, Nora Ephron thought writing was solely about writing.

At high school, she learned that writing, like most things, is much more about thinking.

She tells the story like this:

“My high school journalism teacher, whose name is Charles O Simms, is teaching us to write a lead – the first sentence or paragraph of a newspaper story.

“He writes the words ‘Who What When Where Why and How’ on the blackboard.

“Then he dictates a set of facts to us that goes something like this: ‘Kenneth L Peters, the principal of Beverly Hills High School, announced today that the faculty of the high school will travel to Sacramento on Thursday for a colloquium in new teaching methods. Speaking there will be anthropologist Margaret Mead and Robert Maynard Hutchins, the president of the University of Chicago.’

“We all sit at our typewriters and write a lead, most of us inverting the set of facts so that they read something like this: ‘Anthropologist Margaret Mead and University of Chicago president Robert Maynard Hutchins will address the faculty on Thursday in Sacramento at a colloquium on new teaching methods, the principal of the high school Kenneth L Peters announced today.’ 

We turn in our leads. We’re very proud.

Mr Simms looks at what we’ve done and tosses everything into the garbage.

“He says: ‘The lead to the story is: “**There will be no school on Thursday.”**

What Ephron’s teacher demonstrated to them was that writing isn’t about writing. 

Writing, like everything, is about thinking.





So preparing a presentation is about thinking, not just reducing text for the sake of it.

**Comment by Levijj22** [comment]
Valuable CTF! You can't be a great consultant if you struggle with presentation fundamentals.

**Comment by Marxoz** [comment]
I don't like long presentations, I loose interest really quickly. But well constructed PowerPoint slides can make a massive difference. By having less text might make it a bit better, but it is about how you present the information. 

I see PowerPoints to be used as part of presenting, not something that should be used for a data source. So it shouldn't need so much info that it can be used later without a presenter.

**Comment by ozair-ashfaque** [comment]
Only useful information should be presented on slides with visuals.

**Comment by AshAnil** [comment]
Valuable rule to create a PPT. Designer tool is smart 💡

**Comment by CoffeeBugHub** [comment]
Would like to use SmartArt.

**Comment by Hona** [comment]
Our presentations look way too corporate and therefore become uninteresting. Usage of simplicity & personality would be awesome.

**Comment by Aibono1225** [comment]
Really valuable CTF for creating better slides on PPT

**Comment by camillars** [comment]
Great tips in this CTF - a lot of people have trouble summarizing their talk in just a few words. The rule is a great starting point.

**Comment by JerryLuo5799** [comment]
The tools look so cool, I wil try both of them.

**Comment by jimmidier** [comment]
Very practical tips. A picture says a thousand words, and where text is inevitable, make it animated.

**Comment by sylhuang** [comment]
I tend to use animations with caution. I still remember presentation slides from elementary school where EVERYONE used excessive animations in EVERY slide... including bounce effect for subtitles 🙈

**Comment by johnxu16** [comment]
Great tips! Though it is not always easy to find a suitable image which close related to your topic and memorable at the same time.

**Comment by babakamyljanovssw** [comment]
Great CTF! Designer and SmartArt features will definitely help to create better PPTs

**Comment by SofieHong** [comment]
Great idea! Moreover, combining animation with speech pacing is an art in itself 🎨
✅ Well-executed animation can guide the audience to follow your rhythm and delve deeper
❌ Inappropriate animation may hinder their immersion

**Comment by KristenHu** [comment]
Great topic! Sadly, I never heard these features before, but I'll have a try next time

**Comment by Berber-Mama** [comment]
Agreed with
✅Slides should be your assistant when doing a presentation, do not make it the lead actor.
✅Make the slides less content, try to use animation+pictures.
This approach may require investing more time in preparing your presentation, but it will enhance its overall quality.

**Comment by tino-liu** [comment]
My biology professor's PPT slides were all texts. And he read word for word from those texts. It was pretty amazing nap for quite a lot of students.

**Comment by TobyChurches** [comment]
This is a great rule! These PowerPoint features are a great way to quickly improve the quality of a presentation (even if they are just used for inspiration). I used them at Uni all the time.

**Comment by Freego1783** [comment]
I'm not making Power Point presentations that often but I'll clearly give this a try next time !

**Comment by BrookJeynes** [comment]
I'll have to give these features a go next time I need to create a powerpoint

**Comment by danielmackay** [comment]
Being consultants, the majority of us need to do presentations at some stage (albeit internal or external).  The tips contained in the rule are great points to keep in mind so that we are not causing 'Death by PowerPoint'.  I will keep these in mind for the next slide deck I put together.

I'd be interested to know, did anyone else figure out how to use SmartArt to turn a bulleted list into a timeline as shown in the rule?
